src/java.desktop/macosx/native/libawt_lwawt/awt/AWTWindow.m

```diff
@@ -1024,7 +1024,11 @@ - (void)sendEvent:(NSEvent *)event {
             NSRect contentRect = [self.nsWindow contentRectForFrameRect:frame];
 
             // Check if the click happened in the non-client area (title bar)
-            if (p.y >= (frame.origin.y + contentRect.size.height)) {
+            // Also, non-client area includes the edges at left, right and botton of frame
+            if ((p.y >= (frame.origin.y + contentRect.size.height)) ||
+                (p.x >= (frame.origin.x + contentRect.size.width)) ||
```

Comment:
```suggestion
                (p.x >= (frame.origin.x + contentRect.size.width - 3)) ||
```

From my testing there is a 3 pixel dead zone at the right window border, when the resize cursor is displayed, but `deliverNCMouseDown` is not called.

Please check this on your system as well.

Reply:
Yes, there was a dead zone..Thanks for pointing it out...Updated..